mods/seasonal/moves.js

```diff
@@ -945,6 +1081,34 @@ let BattleMovedex = {
 		type: "Fairy",
 		zMoveEffect: 'clearnegativeboosts',
 	},
+	// MajorBowman
+	blazeofglory: {
+		accuracy: true,
+		basePower: 0,
+		damageCallback: function (pokemon) {
+			let damage = pokemon.hp;
+			pokemon.faint();
+			return damage;
```

Comment:
Did you test this? Not sure if the user faints mid move if it fails.

Reply:
I did, it works. The code was copied from final gambit